### DISCUSSION TITLE
Changelog

### DISCUSSION THREAD

**Discussion by ThinkChaos** [jesec/rtorrent#14] [General]
Hello,

I admire all the work you're doing, and would like to keep myself informed.
Could you provide a short list of changes with each release? Reading all commits takes too much time.

Ideally the changes would be both in a file and on each GH release page.

**Comment by jesec** [comment]
Moving to discussions as this is a request.

**Comment by ThinkChaos** [comment]
Thanks for including a small changelog in r14.

**Comment by silasmariusz** [comment]
Hi

Are the static binaries compiled with IPv6 support?

**Reply by jesec** [reply]
Yes. IPv6 is enabled by default.
